spec/sample-data/valid/control-statement-04.json

```diff
@@ -0,0 +1,84 @@
+{
+  "subject_entity": {
+    "entity_properties": {
+      "company_number": "SL027375",
+      "jurisdiction_code": "gb"
+    },
+    "entity_type": "company"
+  },
+  "sample_date": "2017-06-26",
+  "uid": "/company/SL027375/persons-with-significant-control/individual/P-nakGOmWpZrhD1eVMMwYApUh1U",
+  "source_url": "http://download.companieshouse.gov.uk/en_pscdata.html",
+  "retrieved_at": "2018-01-15T11:38:23+00:00",
+  "confidence": "HIGH",
+  "control_mechanisms": [
+    {
+      "mechanism_type": "right_to_share_surplus_assets",
+      "mechanism_properties": {
+        "exercised_via": null,
+        "percentage_rights": {
+          "minimum": 75,
+          "maximum": 100,
+          "exclusive_minimum": true,
+          "exclusive_maximum": false
+        }
+      },
+      "source_description": "The person holds or is treated as holding, directly or indirectly, the right to share in more than 75% of any surplus assets of the partnership on a winding up."
+    },
+    {
+      "mechanism_type": "right_to_appoint_and_remove_person",
+      "mechanism_properties": {
+        "exercised_via": null
+      },
+      "source_description": "The person holds the right, directly or indirectly, to appoint or remove a majority of the persons who are entitled to take part in the management of the partnership."
+    },
+    {
+      "mechanism_type": "voting_rights",
+      "mechanism_properties": {
+        "exercised_via": null,
+        "voting_percentage": {
+          "minimum": 75,
+          "maximum": 100,
+          "exclusive_minimum": true,
+          "exclusive_maximum": false
+        }
+      },
+      "source_description": "The person holds, directly or indirectly, more than 75% of the voting rights in the company."
+    }
+  ],
+  "controlling_entities": [
+    {
+      "entity_type": "person",
+      "entity_properties": {
+        "date_of_birth": {
+          "month": 7,
+          "year": 1984
+        },
+        "nationality": "Ukrainian",
+        "country_of_residence": "Ukraine",
+        "registered_address": {
+          "country": "United Kingdom",
+          "locality": "Inverness",
+          "postal_code": "IV1 1LD",
+          "region": "Scotland",
+          "street_address": "24 Station Square\nSuite 16"
+        },
+        "name": "Eugen Demchenko",
+        "other_attributes": {
+          "uid": "/company/SL027375/persons-with-significant-control/individual/P-nakGOmWpZrhD1eVMMwYApUh1U",
+          "name_parts": {
+            "given_name": "Eugen",
+            "family_name": "Demchenko"
+          }
+        }
+      }
+    }
+  ],
+  "control_level": "unknown",
+  "ultimate_beneficial_owner": true,
+  "other_attributes": {
+    "etag": "f148cc17ba2e608504f0e9df5e47805034a6a9ad",
+    "ceased_on": null,
+    "notified_on": "2017-06-26"
+  }
+}
```

Comment:
I'm just going to remove this test case. In general I don't think we should add new test cases every time we tweak a schema otherwise it'll just get out of control. We need to be a bit cleverer about it. Aside: I think it would be good to move to _named_ test cases, rather then just numbered (`company-58.json`!), because then at least we can identify what a case is actually aiming to test.